src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -546,11 +546,12 @@ func (c *controller) waitForOLMOperators(ctx context.Context) error {
 			c.log.WithError(err).Error("Timeout while waiting for some of the operators and not able to update its state")
 		}
 	}()
+
 	// Get the monitored operators:
 	c.updateFinalizingProgress(ctx, models.FinalizingStageWaitingForOlmOperatorsCsvInitialization)
 	retryErr = utils.WaitForeverForPredicateWithCancel(ctx, FetchRetryInterval, func() bool {
 		var err2 error
-		operators, err2 = c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID, c.OpenshiftVersion)
+		operators, err2 = c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID, c.OpenshiftVersion, true)
```

Comment:
Should we be passing context.TODO() here or should we update this to pass the context that was passed into the parent function?

Reply:
Why is it important to use cache here?